meshtastic/interfaces/ble/management_service.py

```
@@ -0,0 +1,379 @@
+        management_started = False
+        try:
+            with iface._connect_lock, iface._management_lock:
+                iface._validate_management_preconditions()
+                iface._begin_management_operation_locked()
+                management_started = True
+            target_address = iface._resolve_target_address_for_management(address)
+            canonical_address = iface._format_bluetoothctl_address(target_address)
+            with iface._management_target_gate(target_address):
+                if address is None:
+                    with iface._connect_lock:
+                        with iface._management_lock:
```

Ruff (0.15.5)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)